### PR TITLE
Update roadmap items for R19 Sprint 3

### PR DESCRIPTION
**Pull request by huayunh**
Update for R19 Sprint 3

### CODE REVIEW

File: deployed/doc-it/R19Roadmap.json [hunk 1]
```diff
@@ -446,7 +446,23 @@
...
+        "author": "PX Blue"
+      },
+      {
+        "name": "Updated List and Table Design Pattern",
+        "description": "Update content around the usage of lists and tables",
+        "year": 2021,
+        "quarter": "Q1",
+        "status": "in-progress",
         "author": "PX Blue"
       },
```
Comment (joebochill) [inline deployed/doc-it/R19Roadmap.json:467]: I would leave this off until we see if it's something we will get fully documented this quarter or not. It's not on our original roadmap, so I'm hesitant to commit to it (other than the rough draft for Jonathan) until we get through our other items.

Comment (huayunh) [reply deployed/doc-it/R19Roadmap.json:467]: Yeah makes sense

**Review by emclaug2 — APPROVED** [review]

File: deployed/doc-it/R19Roadmap.json [hunk 2]
```diff
@@ -446,7 +446,15 @@
         "description": "Updated content around the usage of dark themes",
         "year": 2021,
         "quarter": "Q1",
-        "status": "backlog",
+        "status": "in-progress",
+        "author": "PX Blue"
+      },
+      {
+        "name": "User Menu Design Pattern",
+        "description": "Add documentations about the usage of a user menu",
```
Comment (joebochill) [inline deployed/doc-it/R19Roadmap.json:454]:
```suggestion
        "description": "Add documentation about the usage of a user menu",
```

File: deployed/doc-it/R19Roadmap.json [hunk 3]
```diff
@@ -700,7 +708,7 @@
         "description": "Ionic project templates for the CLI",
         "year": 2021,
         "quarter": "Q2",
-        "status": "backlog",
+        "status": "in-progress",
```
Comment (joebochill) [inline deployed/doc-it/R19Roadmap.json:711]: Not sure I would call this in progress yet. We haven't started looking at the CLI at all yet, we're just looking to see if the workflow works with ionic.

**Review by joebochill — APPROVED** [review]

**Pull request merged**